lambda/update/opg_change_status.go

```diff
@@ -0,0 +1,45 @@
+package main
+
+import (
+	"github.com/ministryofjustice/opg-data-lpa-store/internal/shared"
+	"github.com/ministryofjustice/opg-data-lpa-store/internal/validate"
+	"github.com/ministryofjustice/opg-data-lpa-store/lambda/update/parse"
+)
+
+type OpgChangeStatus struct {
+	Status shared.LpaStatus
+}
+
+func (r OpgChangeStatus) Apply(lpa *shared.Lpa) []shared.FieldError {
+
+	if r.Status != shared.LpaStatusCannotRegister {
+		return []shared.FieldError{{Source: "/type", Detail: "Status to be updated should be cannot register"}}
```

Comment:
I believe these should all be status, since that's the JSON pointer to the field that has failed validation:
```suggestion
		return []shared.FieldError{{Source: "/status", Detail: "Status to be updated should be cannot register"}}
```